skbio/diversity/beta/tests/test_unifrac.py

```diff
@@ -8,29 +8,35 @@
 
 from __future__ import absolute_import, division, print_function
 
-from unittest import TestCase, main
 from io import StringIO
-import os
+from unittest import main, TestCase
 
 import numpy as np
-import pandas as pd
+import numpy.testing as npt
 
-from skbio import TreeNode, DistanceMatrix
-from skbio.util import get_data_path
+from skbio import TreeNode
 from skbio.tree import DuplicateNodeError, MissingNodeError
 from skbio.diversity.beta import unweighted_unifrac, weighted_unifrac
+from skbio.diversity.beta._unifrac import (_unweighted_unifrac_pdist_f,
+                                           _weighted_unifrac_pdist_f,
+                                           _unweighted_unifrac,
+                                           _weighted_unifrac,
+                                           _weighted_unifrac_branch_correction)
 
 
-class TestUniFrac(TestCase):
+class UnifracTests(TestCase):
 
     def setUp(self):
-        self.table1 = np.array(
-           [[1, 3, 0, 1, 0],
-            [0, 2, 0, 4, 4],
-            [0, 0, 6, 2, 1],
-            [0, 0, 1, 1, 1],
-            [5, 3, 5, 0, 0],
-            [0, 0, 0, 3, 5]])
+        self.unweighted_unifrac = unweighted_unifrac
```

Comment:
same comment here, is this necessary since there is only a single implementation now?